### PR TITLE
Add `::details-content` example.

### PR DESCRIPTION
**Pull request by lukewarlow**
### Description

Adds an example for the new `::details-content` pseudo element.

### Motivation

To showcase this on the MDN page.

### Additional details

https://drafts.csswg.org/css-pseudo/#details-content-pseudo

### CODE REVIEW

**Comment by lukewarlow** [comment]
Linter doesn't seem aware of this new pseudo element. PR to update stylelint https://github.com/stylelint/stylelint/pull/8174

**Review by chrisdavidmills — APPROVED** [review]
Hi @lukewarlow, thanks for submitting the new example; looks good to me.

I'll approve this, but I'm assuming we shouldn't (or can't) merge it until the new stylelint package is picked up by our code?

**Comment by lukewarlow** [comment]
@chrisdavidmills we can either wait or I can update the stylelint config to ignore the error for this specific pseudo element.

**Comment by chrisdavidmills** [comment]
> @chrisdavidmills we can either wait or I can update the stylelint config to ignore the error for this specific pseudo element.

@lukewarlow I'm happy with either approach. Take your pick.

**Comment by skyclouds2001** [comment]
maybe now it's better to add it to mdn/content

**Comment by lukewarlow** [comment]
Stylelint has been updated so I'm assuming this is ready to go with a rebase (I'm not able to do that till tomorrow but someone else can)? Apologies I let this drop off my radar.

**Comment by bsmth** [comment]
> Stylelint has been updated so I'm assuming this is ready to go with a rebase (I'm not able to do that till tomorrow but someone else can)? Apologies I let this drop off my radar.

Sorry, this is the context -> https://github.com/orgs/mdn/discussions/782#discussion-7920265

tl;dr we're consolidating infrastructure so we can maintain the interactive examples in-page with the rest of the content. See https://github.com/mdn/content/pull/38356/files. If you want to wait for that (`#38356`) PR to land, then we'll accept the changes in the new format in https://github.com/mdn/content/blob/main/files/en-us/web/css/_doublecolon_details-content/index.md?plain=1

__edit:__ and the timeline for "wait for that (`#38356`) PR to land" means tomorrow if all goes to plan

**Comment by bsmth** [comment]
> > Stylelint has been updated so I'm assuming this is ready to go with a rebase (I'm not able to do that till tomorrow but someone else can)? Apologies I let this drop off my radar.
> 
> Sorry, this is the context -> https://github.com/orgs/mdn/discussions/782#discussion-7920265
> 
> tl;dr we're consolidating infrastructure so we can maintain the interactive examples in-page with the rest of the content. See https://github.com/mdn/content/pull/38356/files. If you want to wait for that (`#38356`) PR to land, then we'll accept the changes in the new format in https://github.com/mdn/content/blob/main/files/en-us/web/css/_doublecolon_details-content/index.md?plain=1

__edit:__ and the timeline for "wait for that (`#38356`) PR to land" means tomorrow if all goes to plan

**Comment by lukewarlow** [comment]
Ah I'll move it over there once that's done then!

**Comment by bsmth** [comment]
This is now merged if you'd like to have a go at adding the example in content:

- [x] https://github.com/mdn/content/pull/38356
